<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleColorDlg</class>
 <widget class="QDialog" name="ScaleColorDlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>596</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Color Scales</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Scale File:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QLineEdit" name="txtScaleFile"/>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="cmdScaleFile">
     <property name="text">
      <string>browse ..</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="6">
    <widget class="QLineEdit" name="txtScaleDesc"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Band Count:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="8">
    <widget class="ScaleColorWidget" name="graphicsViewScale">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="cmdDigitize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>Digitize ...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="cmdSave">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="cmdSaveAs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>Save As ...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="cmdClear">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6" colspan="2">
    <widget class="QPushButton" name="cmdClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBandCount">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaleColorWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">scalecolorwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtScaleFile</tabstop>
  <tabstop>cmdScaleFile</tabstop>
  <tabstop>txtScaleDesc</tabstop>
  <tabstop>spinBandCount</tabstop>
  <tabstop>graphicsViewScale</tabstop>
  <tabstop>cmdDigitize</tabstop>
  <tabstop>cmdSave</tabstop>
  <tabstop>cmdSaveAs</tabstop>
  <tabstop>cmdClear</tabstop>
  <tabstop>cmdClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
